docs/functions.rst

```diff
@@ -1572,6 +1572,65 @@ Example:
         args:
           name: "test-snapshot-content-content-dfc8fa67-8b11-4fdf-bf94-928589c2eed8"
 
+BackupDataUsingKopiaServer
+--------------------------
+
+This function backs up data from a container into any object store
+supported by Kanister using Kopia Repository Server as Data-mover.
+
+.. note::
+   It is important that the application includes a ``kanister-tools``
+   sidecar container. This sidecar is necessary to run the
+   tools that capture path on a volume and store it on the object store.
+
+   Additionally, in order to use this function, a RepositoryServer CR is required
+
+Arguments:
+
+.. csv-table::
+   :header: "Argument", "Required", "Type", "Description"
+   :align: left
+   :widths: 5,5,5,15
+
+   `namespace`, Yes, `string`, namespace in which to execute
```

Comment:
`namespace in which to execute`
Maybe we should try to improve this. Users don't know what is being executed where. Even I don't know what is this function to going execute to do what.
If this should be application namespace, let's mention application namespace. Same for the other arguments.

Reply:
Added !